### ISSUE TITLE
[Bug] Scoop powershell in the background using a lot of resources

### ISSUE THREAD

**Issue by Efiyo** [ScoopInstaller/Scoop#5609]
## Bug Report
I noticed my CPU temperature was spiking up and I found that powershell was running on its own for some reason and was performing a scoop search? I have scoop installed with all the default settings, only the install directory is changed.

#### Current Behavior

Scoop randomly started hammering my SSD by performing a search when it was not instructed at any point.

#### Expected Behavior

It should not do that?

#### Additional context/output

Screenshot of the process in Process Hacker that I took before terminating the process.
![image](https://github.com/ScoopInstaller/Scoop/assets/20733843/712333c4-3f41-42e2-a54e-165fa0f33c63)

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
10
**OS architecture:** [e.g. 32bit, 64bit, arm64]
x64
**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
5 1 19041 3031
**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "last_update":  "2023-08-17T06:36:45.6596121+02:00",
    "root_path":  "C:\\Users\\efiyo\\OneDrive\\Applications\\Scoop",
    "global_path":  "C:\\Users\\efiyo\\OneDrive\\Applications\\ScoopGlobal",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by HUMORCE** [comment]
I tried to reproduce the issue but could not:

1. Execute `scoop search` in `pwsh.exe` and `powershell.exe`
2. Check processes `powershell.exe`, `pwsh.exe`, no exceptions
3. Execute `scoop search <random_keyword>` in `pwsh.exe` and `powershell.exe`
4. no exceptions

---

Please provide a minimal reproduction.

**Comment by Efiyo** [comment]
Unfortunately I cannot as I noticed the program running on its own. If I see scoop doing the same thing again I will notify.

**Comment by Efiyo** [comment]
![image](https://github.com/ScoopInstaller/Scoop/assets/20733843/7091a36d-530f-400f-b1f7-fc86fa7b5fcc)

just an update, I rebooted my system and the scoop search started in the background and it is not in the startup apps and is doing the same thing

**Comment by HUMORCE** [comment]
Can you check again with Autoruns? `scoop install extras/sysinternals`

**Comment by Efiyo** [comment]
I got an error that it couldn't find a manifest for that, tried doing `scoop search sysinternals` and I'm getting the same huge I/O of about 75 MB/s from scoop in processes, really hammering it a lot for some reason

**Comment by HUMORCE** [comment]
Download it here: https://learn.microsoft.com/en-us/sysinternals/

---

The [Scoop installer](https://github.com/ScoopInstaller/Install) and [Scoop Core](https://github.com/ScoopInstaller/Scoop) do not add any startup entry, or run in the background.

Check the files of Scoop(Mainly `scoop-search.ps1`):

```
cd C:\Users\efiyo\OneDrive\Applications\Scoop\apps\scoop\current
git status
```

**Comment by Efiyo** [comment]
I understand it shouldn't, but I checked the startup apps with many applications, including the built in and scoop was not there but was executing `scoop search` and eating resources like crazy. I have since unistalled scoop completely and upon reinstallation it works now properly without eating all the resources on search. The point of the issue isn't the startup itself, it's the eating of resources and it not responding to the search command when it was installed with the defaults, no additional buckets added, sans the installation folder which is the only customization.
